README.md

```diff
@@ -22,7 +28,63 @@ Or install it yourself as:
 
 ## Usage
 
-TODO: Write usage instructions here
+The current main usage of `FastStats` is to track and build means while iterating over data sets.
+
+Example: Let's say you want to get some means for social media posts.
+
+You can track several metrics with `FastStats::Means`:
+
+```ruby
+# Say you have some PostFetcher that is an enumerator for your posts.
+means = FastStats::Means.new
+
+PostFetcher.each do |post|
+   # do stuff with post
+   means.add "likes", post["like_count"]
+   means.add "comments", post["share_count"]
+end
+
+means.summary
+# =>
+#{
+#   "foo_arithmetic"=>4.888888888888889,
```

Comment:
Should `foo` and `bar` here be `likes` and `comments` to match the above example?